scripts/borg.android.spec

```diff
@@ -0,0 +1,46 @@
+# -*- mode: python -*-
+# this pyinstaller spec file is used to build borg binaries on posix platforms
```

Comment:
considering we already have another (more generic posix) spec file, maybe rather tell this is about "android" than "posix"?